tools/nl/svindex_differ/differ.py

```diff
@@ -224,6 +224,8 @@ def run_diff(base_file, test_file, base_model_path, test_model_path, query_file,
   for _, diff_table_rows in diffs:
     for row in diff_table_rows:
       for info in row:
+        if not info:
```

Comment:
question: when would this happen?

Reply:
Turns out when getting the diff_table_rows, I wasn't handling a 4th case that the Differ library returns where it returns a "?" which means the line is not present in either input.. fixed